api/schemas/views.py

```diff
@@ -36,10 +34,7 @@ class RegistrationSchemaList(JSONAPIBaseView, generics.ListAPIView, ListFilterMi
     ordering = ('-id',)
 
     def get_default_queryset(self):
-        if waffle.switch_is_active(ENABLE_INACTIVE_SCHEMAS):
-            return RegistrationSchema.objects.get_latest_versions(only_active=False)
-        else:
-            return RegistrationSchema.objects.get_latest_versions()
+        return RegistrationSchema.objects.get_latest_versions_and_allow_egap_admins(self.request, only_active=False)
```

Comment:
Shouldn't the `only_active` parameter value be `True` so that it returns only active RegistrationSchemas and the EGAP schema (assuming a user is an EGAP admin)?

Reply:
I think `only_active` is only set to `False` so it can be filtered by query params by the front end. I think it's more about removing this old special casing, so I reluctant to creep into that, but let's see if I can make act more predictably.

Reply:
You removed `only_active` because it's just handled by the `ListFilterMixin`, correct?

Reply:
yep! it was basically vestigial at this point.